wire-runtime/src/commonTest/kotlin/com/squareup/wire/ProtoReaderTest.kt

```diff
@@ -32,4 +32,25 @@ class ProtoReaderTest {
     assertEquals(-1, reader.nextTag())
     reader.endMessageAndGetUnknownFields(token)
   }
+
+  @Test fun lengthDelimited() {
+    val encoded = "0208020608ffffffff07".decodeHex()
```

Comment:
```suggestion
    // Bytes representing:
    // ── 0 ┐
    //      ╰- 0 ┐
    //           ╰- 1: 2147483647
    val encoded = "0208020608ffffffff07".decodeHex()
```

Reply:
This is not a single proto. It's two protos prefixed with their varint32-encoded lengths. I'll add a comment.

Reply:
sweet